docs/configuration/configuration.md

```diff
@@ -1724,6 +1724,9 @@ url: <string>
 headers:
   [ <string>: <string> ... ]
 
+# Retry on receiving a rate-limiting status code in response from remote-write storage.
```

Comment:
Would you put a note here saying that this configuration may change in any release, similar to the note above `metadata_config`. I imagine we may get rid of it once there is an agreed upon solution to how to handle rate limited retries.

Reply:
HTTP 429 should be mentioned.

I am not confortable to change this behavior in a minor release.

Reply:
This doc string should also say retry on 429 status code, and the variable name two lines down still needs to be updated.

Reply:
variable name?

Reply:
`retry_on_rate_limit` should be `retry_on_http_429` now a couple lines below this.

Reply:
Done, I hope the message is good.